CK3toEU4/Data_Files/configurables/religion_map.txt

```diff
@@ -1,7 +1,7 @@
 # Usage:
 # link = { eu4 = eu4religion ck3 = ck3faith }
 # multiple ck3 entires are allowed (and encouraged), multiple eu4 are not.
-# A special entry "school" can be used when mapping to sunni or shiite, specifying
+# A special entry "school" can be used when mapping to Muslim religions, specifying
```

Comment:
Only Sunni and Shia have schools of law, other Islamic denominations don't

Reply:
Schools of law is not suuni/shia specific. It is for all religions in the muslim group. Ibadi has them as well in the base game.